unit-tests/src/test/java/org/eclipse/collections/impl/AbstractRichIterableTestCase.java

```diff
@@ -1491,7 +1491,7 @@ public void chunk_single()
     {
         RichIterable<String> collection = this.newWith("1");
         RichIterable<RichIterable<String>> groups = collection.chunk(2);
-        Assert.assertEquals(FastList.newListWith(1), groups.collect(RichIterable::size));
+        Assert.assertEquals(Lists.mutable.with(1), groups.collect(RichIterable::size, Lists.mutable.empty()));
```

Comment:
The Stack implementations return a different type for collect.  I understood this method to be testing that the result size should be 1 even though the chunk size requested was 2.  I didn't think that testing the return type of collect was a specifically a List was necessary for this specific test.  I can revert and override this test specifically for Stacks.